assets/app/view/game/entities.rb

```diff
@@ -47,13 +47,19 @@ def render
           h(Bank, game: @game),
           h(GameInfo, game: @game, layout: 'upcoming_trains'),
           *@game.corporations.select(&:receivership?).map { |c| h(Corporation, corporation: c) },
+          *unowned_companies,
           *extra_bank,
         ].compact)
 
         children.concat(bankrupt_players.map { |p| h(:div, [h(Player, player: p, game: @game)]) })
 
         h('div#entities', div_props, children)
       end
+
+      def unowned_companies
+        companies = @game.unowned_purchasable_companies(@current_entity)
```

Comment:
Is @current_entity defined in this context?

Reply:
yes, it's a player or a corporation